control/server.py

```diff
@@ -127,6 +128,9 @@ def __exit__(self, exc_type, exc_value, traceback):
             logger.info("Exiting the gateway process.")
         gw_logger.compress_final_log_file(gw_name)
 
+        if self.status_conn: 
+            self.status_conn.shutdown()
```

Comment:
Better use the existing connection which we already shutdown. So this code is not needed